openff/toolkit/_tests/test_topology.py

```diff
@@ -780,6 +781,10 @@ def test_from_pdb_input_types(self):
         with open(protein_path) as f:
             Topology.from_pdb(f)
 
+        pdb_string = pathlib.Path(protein_path).read_text()
+        with StringIO(pdb_string) as f:
+            Topology.from_pdb(f)
+
```

Comment:
Would a trick like you did here also work? I stole it for use elsewhere so I'm curious if there is nuance outside of "if it works, it works"

https://github.com/openforcefield/openff-toolkit/blob/d0e63b9dbb540bf8a9a6d9aa48921f0c01d257b4/openff/toolkit/topology/topology.py#L2035-L2039